### ISSUE TITLE
Trigger a Jenkins job clicking on the 'enable env' button after a deploy

### ISSUE THREAD

**Issue by NotoriousBig** [vertigo17/Cerberus#923]
Hello Guys,
 
For now we have a method to launch a a cerberus campaign right after a deploy in dev. But it is not possible for any env outside the dev.
 
I’d like to set up the following configuration 
•	DEV – deploy application – run application related campaign
•	QA/UAT – deploy system – run system related campaign
 
It’s already OK for dev as we can launch campaign ‘myApp-DEV’.
But this is non-sense for QA/UAT as we deploy system, not only application and we want to launch tests related to the target env (functionnal tests in QA, technical test in RE7…)
 
My proposal on that point is to uptade the following buttons so it triggers the dedicated campaign :
 
![screen](https://cloud.githubusercontent.com/assets/13114629/18578835/78a5ef14-7bf2-11e6-9b39-8ec8d56285a7.png)

The best way to me is the button calling jenkins a jenkins job related to the deploy/test campaign and  jenkins asking Cerberus to launch the campaign. 

Regards
Seb

**Comment by abourdon** [comment]
@bcivel @vertigo17 : I wonder if this feature will be interesting. What do you think?

**Comment by NotoriousBig** [comment]
Hello,  as a reminder, I requested this update for the following reason :

Usually we build test campaign to test our application. Campaign can be
launched from outside Cerberus (Jenkins for instance)
We might want to launch a campaign right after the deploy on target
environment.  Today this is done manually, but this need coordination
betwen deploy team and test team.

The main purpose of this request is to be able to trigger the jenkins job
that trigger the campaign with no other action than deploying.

The 'enable env' button is the good place to do that because you ensure the
full build/rev is deployed and you ensure the env is enabled also

regards
Séb

2017-03-29 14:14 GMT+02:00 Aurélien Bourdon <notifications@github.com>:

**Comment by vertigo17** [comment]
As Sebastien explained, for me the feature still make sense but I would prefer it gets sponsored by the deploy team.
The only comment I could have is that the trigger to the campaign execution should be done only on the modal on the button 'Enable and Sent Notification'. This is the place where all deploys are done on every application of the system.

**Comment by abourdon** [comment]
Yes for me it is just an integration issue. Cerberus already have API to start a campaign with any configuration parameter to satisfy request. Cerberus should not manage Jenkins but Jenkins should be able to manage Jenkins

I propose to close this one as this is not a dedicated Cerberus issue.

Let me know

**Comment by vertigo17** [comment]
Well you are right as long as deploy is clean and controlled at the application level.
Sometimes we have to deploy a lot of different application before beeing able to trigger the campaing. Those application are dependant each other.
We use Cerberus for that using the environment screen that allow to identify all application changes (grouped within a system) and decide wich version to deploy on each application.
In that case, Cerberus becomes an Integration tool and it make sense to plug that Cerberus-Integration to trigger Cerberus-Testing (still using the APi you mentioned).
